runtime/battery-station/src/integration_tests/xcm/setup.rs

```diff
@@ -0,0 +1,210 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+// Copyright 2022 Forecasting Technologies LTD.
+//
+// This file is part of Zeitgeist.
+//
+// Zeitgeist is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at
+// your option) any later version.
+//
+// Zeitgeist is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Zeitgeist. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{
+    xcm_config::{
+        asset_registry::CustomMetadata,
+        config::{battery_station, general_key},
+    },
+    AccountId, AssetRegistry, Balance, CurrencyId, ExistentialDeposit, Origin, Runtime, System,
+};
+use frame_support::{assert_ok, traits::GenesisBuild};
+use orml_traits::asset_registry::AssetMetadata;
+use xcm::{
+    latest::{Junction::Parachain, Junctions::X2, MultiLocation},
+    VersionedMultiLocation,
+};
+use zeitgeist_primitives::types::Asset;
+
+pub(super) struct ExtBuilder {
+    balances: Vec<(AccountId, CurrencyId, Balance)>,
+    parachain_id: u32,
+}
+
+impl Default for ExtBuilder {
+    fn default() -> Self {
+        Self { balances: vec![], parachain_id: battery_station::ID }
+    }
+}
+
+impl ExtBuilder {
+    pub fn balances(mut self, balances: Vec<(AccountId, CurrencyId, Balance)>) -> Self {
```

Comment:
should we name this set_balances() ? and same for parachain_id below?

Reply:
Applied in https://github.com/zeitgeistpm/zeitgeist/pull/661/commits/7ebe84aff0b32e56910207031bc5d7d7399ba123